api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+			"type": "object",
+			"properties": {
+				"mutable": {
+                                        "description": "Changeable configuration",
+                                        "$ref": "#/definitions/ConfigurationMap"
+				},
+				"immutable": {
+                                        "description": "Immutable configuration (read-only)",
+                                        "$ref": "#/definitions/ConfigurationMap"
+				}
+			}
+		},
+		"ConfigurationMap": {
+                        "description": "List of configuration settings",
+			"type": "object",
+			"additionalProperties": {
+				"type": "string"
+			}
+		},
+		"NodeAddressing": {
+			"description": "Addressing information of a node",
+                        "type": "object",
+			"properties": {
+				"ipv6": {
+					"$ref": "#/definitions/NodeAddressingElement"
+				},
+				"ipv4": {
+					"$ref": "#/definitions/NodeAddressingElement"
+				}
+			}
+		},
+		"NodeAddressingElement": {
+			"description": "Addressing information element",
+                        "type": "object",
+			"properties": {
+				"enabled": {
+					"description": "Address family is enabled",
+					"type": "boolean"
+				},
+				"ip": {
+                                        "description": "IP address of node",
+                                        "type": "string"
+				},
+				"alloc-range": {
+                                        "description": "Gateway address of containers",
+                                        "type": "string"
+				}
+			}
+		},
+		"PolicyTree": {
+			"description": "Policy tree or subtree",
+			"type": "string"
+		},
+		"PolicyTraceResult": {
+			"description": "Response to a policy resolution process",
+			"type": "object",
+			"properties": {
+				"verdict": {
+					"type": "string"
+				},
+				"log": {
+					"type": "string"
+				}
+			}
+		},
+		"IdentityContext": {
+			"description": "Context describing a pair of source and destination identies",
+			"type": "object",
+			"properties": {
+				"from": {
+					"$ref": "#/definitions/Labels"
+				},
+				"to": {
+					"$ref": "#/definitions/Labels"
+				}
+			}
+		},
+		"FrontendAddress": {
+			"description": "Layer 4 address",
+                        "type": "object",
+			"required": [ "ip" ],
+			"properties": {
+				"ip": {
+					"description": "Layer 3 address",
+					"type": "string"
+				},
+				"protocol": {
+					"description": "Layer 4 protocol",
+					"type": "string",
+					"enum": [
+						"tcp",
+						"udp",
+						"any"
+					]
+				},
+				"port": {
+					"description": "Layer 4 port number",
+					"type": "integer",
+                                        "format": "uint16"
+				}
+			}
+		},
+		"BackendAddress": {
+			"description": "Service backend address",
+                        "type": "object",
+			"required": [ "ip" ],
+			"properties": {
+				"ip": {
+					"description": "Layer 3 address",
+					"type": "string"
```

Comment:
Do we need to make this more fine-grained here and in "FrontendAddress" wrt differentiating ipv4/ipv6?